### ISSUE TITLE
Torrentz and ExtraTorrent links

### ISSUE THREAD

**Issue by viorel-m** [DiMartinoXBMC/plugin.video.torrenter#53]
hi, when trying to download or open torrents for torrentz and extratorrent links, the following error occur :
```
20:33:08 27596.042969 T:1285854112  NOTICE: ### [Torrenter v.2.5.2g]: Imported libtorrent v1.1.0.0 from python_libtorrent
20:33:13 27601.048828 T:1285854112   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.AttributeError'>
                                            Error Contents: SkorbaLoader instance has no attribute 'torrentFile'
                                            Traceback (most recent call last):
                                              File "/storage/.kodi/addons/plugin.video.torrenter/default.py", line 41, in <module>
                                                core.executeAction(params)
                                              File "/storage/.kodi/addons/plugin.video.torrenter/Core.py", line 1148, in executeAction
                                                getattr(self, get("action"))(params)
                                              File "/storage/.kodi/addons/plugin.video.torrenter/Core.py", line 1478, in openTorrent
                                                self.__settings__.setSetting("lastTorrent", torrent.saveTorrent(url))
                                              File "/storage/.kodi/addons/plugin.video.torrenter/Downloader.py", line 65, in saveTorrent
                                                return self.player.saveTorrent(torrentUrl)
                                              File "/storage/.kodi/addons/plugin.video.torrenter/SkorbaLoader.py", line 91, in saveTorrent
                                                return self.torrentFile
                                            AttributeError: SkorbaLoader instance has no attribute 'torrentFile'
                                            -->End of Python script error report<--
20:33:14 27601.292969 T:1963012096   ERROR: GetDirectory - Error getting plugin://plugin.video.torrenter/?action=openTorrent&url=Torrentz%3A%3Amagnet%3A%3Fxt%3Durn%3Abtih%3Ad8c5dcd9dc68c1f490c093696085873eb2d79dc8%26dn%3Drise%2Bof%2Bnations%2Bthrones%2Band%2Bpatriots%2Bworks%2Bonline%2Bofficial%2Bse%26tr%3Dudp%253A%252F%252Ftracker.openbittorrent.com%253A80%252Fannounce%26tr%3Dudp%253A%252F%252Fopen.demonii.com%253A1337
20:33:14 27601.294922 T:1963012096   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.torrenter/?action=openTorrent&url=Torrentz%3A%3Amagnet%3A%3Fxt%3Durn%3Abtih%3Ad8c5dcd9dc68c1f490c093696085873eb2d79dc8%26dn%3Drise%2Bof%2Bnations%2Bthrones%2Band%2Bpatriots%2Bworks%2Bonline%2Bofficial%2Bse%26tr%3Dudp%253A%252F%252Ftracker.openbittorrent.com%253A80%252Fannounce%26tr%3Dudp%253A%252F%252Fopen.demonii.com%253A1337) failed
```

**Comment by viorel-m** [comment]
This issue is fixed when applied @Palatosino 's #52  pull request

**Comment by DiMartinoXBMC** [comment]
update in repo
